Lib/compression/zstd/_zstdfile.py

```diff
@@ -0,0 +1,378 @@
+import builtins
```

Comment:
We're already importing `io`, so we can use `io.open` as ``builtins.open is io.open``.
```suggestion
```